pkg/skaffold/build/cache/hash_test.go

```diff
@@ -47,6 +47,8 @@ var fakeArtifactConfig = func(a *latest.Artifact) (string, error) {
 	return "", nil
 }
 
+const Dockerfile = "Dockerfile"
```

Comment:
This doesn't seem to save much :-)

Reply:
we are at the mercy of linters 🤷🏾‍♂️

Reply:
_"We shape our tools, and then our tools shape us"_